File: heyborsa-spa/src/app/entities/Question.ts
export class Question implements IEntity
{

	constructor($id: Number, $user_id: Number, $question_header: String, $question: String, $question_status: Number, $date: Date,$subject: String) {
		this.id = $id;
		this.user_id = $user_id;
		this.question_header = $question_header;
		this.question = $question;
		this.question_status = $question_status;
		this.date = $date;
		this.subject = $subject;
	}
    public id:Number;
	public user_id:Number;
	public subject:String;
    public question_header:String;
    public question:String;
    public question_status:Number;
    public date:Date;
}

File: heyborsa-api/src/main/java/com/heyborsa/entity/Question.java
package com.heyborsa.entity;

import java.util.Date;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
@Entity
public class Question {
	@Id
	@GeneratedValue(strategy=GenerationType.IDENTITY)
	private long id;
	private long user_id;
	private Date date;
	private String subject;
	private String header;
	private String content;
	private String status;
	
	
	
	public Question(long id, long user_id, Date date, String subject, String header, String content, String status) {
		super();
		this.id = id;
		this.user_id = user_id;
		this.date = date;
		this.subject = subject;
		this.header = header;
		this.content = content;
		this.status = status;
	}



	public long getId() {
		return id;
	}



	public void setId(long id) {
		this.id = id;
	}



	public long getUser_id() {
		return user_id;
	}



	public void setUser_id(long user_id) {
		this.user_id = user_id;
	}



	public Date getDate() {
		return date;
	}



	public void setDate(Date date) {
		this.date = date;
	}



	public String getSubject() {
		return subject;
	}



	public void setSubject(String subject) {
		this.subject = subject;
	}



	public String getHeader() {
		return header;
	}



	public void setHeader(String header) {
		this.header = header;
	}



	public String getContent() {
		return content;
	}



	public void setContent(String content) {
		this.content = content;
	}



	public String getStatus() {
		return status;
	}



	public void setStatus(String status) {
		this.status = status;
	}



	public Question()
	{
		
	}

}
